projects/disagreement/run.py

```diff
@@ -195,7 +199,7 @@ def make_env_all_params(rank, args):
         # TODO: Resume wandb logging is not working
         run = wandb.init(
             project="embodiedAI",
-            name=run_args.exp_name,
+            name=run_args.exp_name + logging_args.project_id,
```

Comment:
To follow the pattern of the checkpoint above, shouldn't we also add an underscore between exp_name and id? Or conversely remove the underscore from the previous. Any of those two would make them consistent.

Reply:
Right, good point, I added an underscore here.